bin/main.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
Adding it here so it shows up inline: move the main executable to the root directory and give it another name such as vsm, so that it can live in /usr/sbin or such.